docs/docs/guides/how-to-create-alerts.mdx

```diff
@@ -0,0 +1,95 @@
+import ThreeDotMenu from './assets/alert-three-dot-menu.png';
+import CreateNewModal from './assets/alert-create-new-modal.png';
+import ConfigureNewAlert from './assets/configure-alert.png';
+import EditDeleteAlert from './assets/edit-delete-alert.png';
+import PrivateSlackChannel from './assets/private-slack-channel.png';
+
+# Alerts
+
+Alerts enable you to receive notifications via email or Slack when the value of a saved chart meets your specified conditions.
+
+If your Lightdash organization has email or [Slack](/guides/sharing-in-slack.mdx#adding-slack-to-your-organization-lightdash-cloud) set up, you can create or select an existing saved chart or dashboard, add an alert to it, and tell Lightdash when you’d like the send out your update. You can set up as many alerts to a saved chart as you like, and if you make any changes to the content, Lightdash will update the alert the next time they’re sent.
```

Comment:
Maybe we should be a little more clear here when is happening when an alert is sent. So it's clear the time is when the query will happen. 

Or maybe a "how it works" section? All it really has to say is something like

"At the scheduled time, Lightdash will run the query and check the results if the first value in the results set meets the alerting condition, the alert will be sent."

And maybe add something about how it will continue to be sent if the condition is true until it is disabled?